src/conn/conn_api.c

```diff
@@ -1109,6 +1109,28 @@ __conn_config_append(const char *cfg[], const char *config)
 }
 
 /*
+ * __conn_config_readonly --
+ *	Append an entry to a config stack that overrides some settings
+ *	when read-only is configured.
+ */
+static void
+__conn_config_readonly(const char *cfg[])
+{
+	const char *readonly;
+
+	/*
+	 * Override certain settings.  Other settings at odds will return
```

Comment:
Can we better define "certain" here? Your comment below makes it sound as though the intent is for this string to cover cases where the default settings would conflict - is that a reasonable general statement?